### PR TITLE
Draft application: working tokens/accounts/transactions/errors via berlingroup api

### PR DESCRIPTION
**Pull request by connicov1**

### CODE REVIEW

File: ruby-connector-v2-sample/app/jobs/callbacks/session_successer.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
+class Callbacks::SessionSuccesser < ApplicationJob
```
Comment (ColVic) [inline ruby-connector-v2-sample/app/jobs/callbacks/session_successer.rb:1]: I suggest a better naming, something like ` Callbacks::SessionSuccess`. Same for Updater.

Comment (mtkachenk0) [reply ruby-connector-v2-sample/app/jobs/callbacks/session_successer.rb:1]: I prefer using nouns for class namings and verbs for methods

File: ruby-connector-v2-sample/app/controllers/api/priora/v2/accounts_controller.rb [hunk 2]
```diff
@@ -5,6 +5,6 @@ def index
 
   def transactions
     # NOTE: validate and serialize
-    render json: {data: current_token.user.accounts.find_by(id: params[:account_id])&.transactions || []}
+    render json: {data: current_token.user.accounts.find_by(id: jwt_params[:account_id])&.transactions || []}
```
Comment (mtkachenk0) [inline ruby-connector-v2-sample/app/controllers/api/priora/v2/accounts_controller.rb]: account_id is inside params

File: ruby-connector-v2-sample/app/controllers/api/priora/v2/base_controller.rb [hunk 3]
```diff
@@ -8,7 +8,7 @@ class Api::Priora::V2::BaseController < ApplicationController
   rescue_from StandardError, with: :internal_server_error
   rescue_from ApiError, with: :api_error
 
-  def decode_jwt
+  def jwt_params
     params.merge! JWT.decode(bearer_token, PRIORA_PUBLIC_KEY, true, {algorithm: 'RS256'}).first["data"]
```
Comment (mtkachenk0) [inline ruby-connector-v2-sample/app/controllers/api/priora/v2/base_controller.rb]: @jwt_params ||= ...

File: ruby-connector-v2-sample/app/jobs/application_job.rb [hunk 4]
```diff
@@ -4,6 +4,6 @@ class ApplicationJob < ActiveJob::Base
   queue_as :default
 
   rescue_from StandardError do |ex|
-    binding.pry
+    Rails.logger.warn error(ex.message)
```
Comment (mtkachenk0) [inline ruby-connector-v2-sample/app/jobs/application_job.rb]: `Rails.logger.warn "#{ex.class}: #{ex.message}"`

**Pull request merged**